.github/workflows/reusable/setup.yml

```
@@ -3,12 +3,12 @@ name: Build
 on:
   workflow_call:
     inputs:
-		node-version:
-			description: Version of Node to use
-			default: "20.x"
-		pnpm-version:
-			description: Version of pnpm to use
-			default: "10.8.0"
+      node-version:
```

actionlint (1.7.12)

[error] "type" is missing at "node-version" input of workflow_call event

(syntax-check)